.github/workflows/linux-32bit-build-and-test.yaml

```
@@ -55,29 +63,56 @@ jobs:
 
     - name: make installcheck
       id: installcheck
+      shell: bash
       run: |
         set -o pipefail
+        export LANG=C.UTF-8
         sudo -u postgres make -k -C build installcheck IGNORES="${{ matrix.ignores }}" | tee installcheck.log
 
     - name: Show regression diffs
       if: always()
       id: collectlogs
+      shell: bash
       run: |
         find . -name regression.diffs -exec cat {} + > regression.log
         find . -name postmaster.log -exec cat {} + > postgres.log
         grep -e 'FAILED' -e 'failed (ignored)' installcheck.log || true
+        if [[ -s regression.log ]]; then echo "::set-output name=regression_diff::true"; fi
         cat regression.log
 
-    - name: Save regression diffs
+    - name: Coredumps
       if: always()
+      id: coredumps
+      shell: bash
+      run: |
+        if compgen -G "/tmp/core*" > /dev/null; then
+          apt-get install postgresql-${{ matrix.pg }}-dbgsym >>$LOGFILE
+          for file in $(ls /tmp/core*)
+          do
```

Given that you ran `compgen` before, you know that it will expand to something.
```suggestion
          for file in ls /tmp/core*; do
```